quantiles/include/quantiles_sketch.hpp

```diff
@@ -504,21 +481,25 @@ class quantiles_sketch {
   static const size_t DATA_START = 16;
 
   Allocator allocator_;
+  bool is_level_zero_sorted_;
```

Comment:
if renaming this, it needs to be `is_base_buffer_sorted_` -- it is not levels_[0] but a different structure.